src/client/components/ShoppingItem/ShoppingItem.js

```diff
@@ -47,7 +52,7 @@ export default function ShoppingItem({
         <Numberinput
           maxAvailable={quantity}
           getQuantity={setItemValue}
-          initValue={initValue}
+          initValue={itemValue}
           disabled={isDisable}
         />
         <p style={{ color: textColor }}>{itemValue * price} DKK</p>
```

Comment:
i think : instead of putting `subTotal` value on the `price` section, you can may be add another line for subTotal value as

        <p style={{ color: textColor }}>{price} DKK</p>
        <p style={{ color: textColor }}>SubTotal: {itemValue * price} DKK</p>

Reply:
that is not in my scope, that is Semih change.. I don't want touch too much component code.